---
layout: workshop_main_3day
permalink: /RNAseq_2019
title: RNA-Seq
header1: Workshop Pages for Students
header2: Informatics for RNA-Seq Analysis 2019
image: /site_images/CBW_RNA_seq_icon.jpg
keywords: RNA-seq data, differential expression analysis, assemble transcripts
description: Course providing an introduction to RNA-seq data analysis followed by integrated tutorials demonstrating the use of popular RNA-seq analysis packages.
instructors: Malachi Griffith, Obi Griffith, Brian Haas, Kelsy Cotto, Megan Richters
length: 3 days
---
# Welcome <a id="welcome"></a>

Welcome to Informatics for RNA-Seq Analysis 2019.

The course schedule can be found [here](https://bioinformaticsdotca.github.io/RNAseq_2019_schedule).

We are using Slack for discussion. Join the Slack channel.

CLASS MATERIALS: The full course notes are available as a [single PDF]().  

# Pre-Workshop Preparation <a id="preworkshop"></a>

## Laptop Setup Instructions

Instructions to setup your laptop can be found [here](https://bioinformaticsdotca.github.io/RNAseq_laptop_setup_instructions).

## Pre-workshop Tutorials

Pre-readings and pre-work can be found here [here](https://bioinformaticsdotca.github.io/RNAseq_2019_prework).  
It is in your best interest to complete these before the workshop.

***

# Class Photo
 
Coming soon

***

# Day 1 <a id="day1"></a>

**Day 1 and Day 2 refer to content in the [RNA-Seq wiki](https://github.com/griffithlab/rnaseq_tutorial/wiki).**

##  Welcome 

  *<font color="#827e9c">Michelle Brazas</font>* 

## Module 1: Introduction to Cloud Computing

*<font color="#827e9c">Malachi Griffith</font>* 

[Lecture]()  

[Lab Practical: Connecting to the Cloud](http://bioinformaticsdotca.github.io/AWS_setup)

* We have set up instances on the Amazon cloud - one for each student. In order to log in to your instance, you will need a security certificate. If you plan on using Linux or Mac OS X, please download [this certificate](). Otherwise if you plan on using Windows (with Putty), please download [this certificate]().  

## Module 2: Introduction to RNA Sequencing Analysis

*<font color="#827e9c">Malachi Griffith</font>* 

[Lecture]()   

### Module 2 Lab Practical

[Lab practical]()

  [Installation](https://github.com/griffithlab/rnaseq_tutorial/wiki/Installation)
  
  [Reference Genomes](https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Genome)
  
  [Annotations](https://github.com/griffithlab/rnaseq_tutorial/wiki/Annotation)
  
  [Indexing](https://github.com/griffithlab/rnaseq_tutorial/wiki/Indexing)
  
  [Data](https://github.com/griffithlab/rnaseq_tutorial/wiki/RNAseq-Data)
  
  [Data QC](https://github.com/griffithlab/rnaseq_tutorial/wiki/PreAlignment-QC)


## Module 3: RNA-seq Alignment and Visualization

*<font color="#827e9c">Malachi Griffith</font>* 

[Lecture]()

## Module 3: RNA-seq Alignment and Visualization Lab Practical

*<font color="#827e9c">Malachi Griffith</font>* 

[Lab Practical](https://drive.google.com/open?id=1CgEtVNY-B3-p0j-4CjGNvU0iwFetZlgE)

  [Adapter Trim](https://github.com/griffithlab/rnaseq_tutorial/wiki/Adapter-Trim)
  
  [Alignment](https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment)
  
  [IGV](https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial)

  [Alignment Visualization](https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization)
  
  [Alignment QC](https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-QC)


## Integrated Assignment

*<font color="827e9c">Megan Richters and Kelsy Cotto</font>*

[Assignment Questions](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment)

[Assignment Answers](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment-Answers)  


***

# Day 2 <a id="day2"></a>


## Module 4: Expression and Differential Expression

*<font color="#827e9c">Obi Griffith</font>* 

[Lecture]()
 
## Module 4 Lab

[Lab Practical]()

 [Expression](https://github.com/griffithlab/rnaseq_tutorial/wiki/Expression)
 
 [Differential Expression](https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Expression)
 
 [DE Visualization](https://github.com/griffithlab/rnaseq_tutorial/wiki/DE-Visualization)

## Module 5: Reference Free Alignment

*<font color="#827e9c">Obi Griffith</font>* 

[Kallisto](https://github.com/griffithlab/rnaseq_tutorial/wiki/Kallisto)  

## Module 6: Isoform Discovery and Alternative Expression

*<font color="#827e9c">Obi Griffith</font>* 

[Lecture]()

## Integrated Assignment

[Assignment Questions](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment)  

[Assignment Answers](https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment-Answers)  

## Keeping Up-to-date with RNA-Seq Analysis Developments

For additional resources, tutorials, future directions, and more please refer to the [RNA-seq wiki](http://www.rnaseq.wiki/)

***

# Day 3 <a id="day3"></a>

## Module 7: Genome Guided and Genome-Free Transcriptome Assembly

*<font color="#827e9c">Brian Haas</font>* 

[Lecture]()  

[Bandage](https://rrwick.github.io/Bandage/)  

[Lab practical](https://bioinformaticsdotca.github.io/rnaseq_2018_tutorial5)  


## Module 8: Functional Annotation and Analysis of Transcripts

*<font color="#827e9c">Brian Haas</font>* 

[Lecture](https://drive.google.com/open?id=1b1opRzV_IK4QSJHXo-B-iQvBFenV_23h)  

[Lab practical](https://bioinformaticsdotca.github.io/rnaseq_2018_tutorial6)  

***  

## Additional Optional Resources 

Additional resources can be found on the [RNA-Seq wiki](https://github.com/griffithlab/rnaseq_tutorial/wiki/Resources).

***

Thank you for attending the Informatics for RNA-Seq Analysis workshop! Help us make this workshop better by filling out [our survey]()

***
  
