---
layout: tutorial_page
permalink: /RNAseq_2019_prework
title: RNA-Seq
header1: Workshop Pages for Students
header2: Informatics for RNA-Seq Analysis 2019
image: /site_images/CBW_RNA_seq_icon.jpg
home: https://bioinformaticsdotca.github.io/RNAseq_2019
---

## Do these before coming to the workshop:

1) **R Preparation tutorials**: You are expected to be familiar with **R** before the workshop. If you are unfamiliar with **R**, here are some suggested tutorials:  

* The [CBW R tutorial](http://bioinformatics-ca.github.io/CBW_R_Tutorial/) or [R Tutorial](http://www.cyclismo.org/tutorial/R/) 
* The [R command cheat sheet](https://github.com/bioinformaticsdotca/bioinformaticsdotca.github.io/blob/master/resources/R_Short-refcard.pdf)
* [R Plotting Reference](https://github.com/bioinformatics-ca/bioinformatics-ca.github.io/blob/master/resources/Plotting.Reference.ipynb)

2) **UNIX Preparation tutorials**: You are expected to be familiar with basic command line usage before the workshop. If you are unfamiliar with **the command line**, here are some suggested tutorials:   

* [UNIX Bootcamp](https://rnabio.org/module-00-setup/0000/06/01/Unix/)
* [UNIX Bootcamp alternative](http://rik.smith-unna.com/command_line_bootcamp/?id=9xnbkx6eaof) 
* [Unix Cheat sheet](http://www.rain.org/~mkummel/unix.html) 

3) [Sequencing Terminology](http://www.ncbi.nlm.nih.gov/projects/genome/glossary.shtml)


## Read these before coming to the workshop:

* [Informatics for RNA Sequencing: A Web Resource for Analysis on the Cloud](https://www.ncbi.nlm.nih.gov/pubmed/26248053)  

* [Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration](http://www.ncbi.nlm.nih.gov/pubmed/22517427)

* [Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown](https://www.ncbi.nlm.nih.gov/pubmed/27560171)
  
* [ENCODE RNA-Seq Standards](https://genome.ucsc.edu/ENCODE/protocols/dataStandards/ENCODE_RNAseq_Standards_V1.0.pdf)
  
* [Methods to study splicing from high-throughput RNA sequencing data](http://www.ncbi.nlm.nih.gov/pubmed/24549677)  
  
* [A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control Consortium](http://www.ncbi.nlm.nih.gov/pubmed/25150838)

* [De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis](http://www.nature.com/nprot/journal/v8/n8/full/nprot.2013.084.html)


## Optional readings:

* [HISAT: a fast spliced aligner with low memory requirements](https://www.ncbi.nlm.nih.gov/pubmed/25751142)  

* [StringTie enables improved reconstruction of a transcriptome from RNA-seq reads](https://www.ncbi.nlm.nih.gov/pubmed/25690850)  

* [Ballgown bridges the gap between transcriptome assembly and expression analysis](https://www.ncbi.nlm.nih.gov/pubmed/25748911)  

* [HTSeq--a Python framework to work with high-throughput sequencing data](https://www.ncbi.nlm.nih.gov/pubmed/25260700)  

* [edgeR: a Bioconductor package for differential expression analysis of digital gene expression data](https://www.ncbi.nlm.nih.gov/pubmed/19910308)  

* [Full-length transcriptome assembly from RNA-Seq data without a reference genome](http://www.nature.com/nbt/journal/v29/n7/abs/nbt.1883.html)  

